server/tools/hatohol-house-keeper

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+import sys
+import os
+import argparse
+import MySQLdb
+import time
+
+
+def finish_if_require_option_is_fault(args):
```

Comment:
The sentence "option is fault" doesn't makes sense.
Do you mean  "finish_if_required_option_is_missing"